src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/gerritnotifier/ParameterExpander.java

```diff
@@ -104,11 +109,18 @@ public String getBuildStartedCommand(Run r, TaskListener taskListener,
             ChangeBasedEvent event, BuildsStartedStats stats) {
 
         GerritTrigger trigger = GerritTrigger.getTrigger(r.getParent());
+
+        Map<String, Integer> labelVotes = new HashMap<>();
+        labelVotes.put("CODE_REVIEW", getBuildStartedLabelValue(trigger.getCodeReviewLabel()));
```

Comment:
Is there any reason for these two labels to be handled separately?

Reply:
Done.